index.html

```diff
@@ -8,29 +8,167 @@
         </p>
-        <img src="images/Vector 4.png" class="line" alt="Horizontal line">
-       </section>
-   </section>
+      </div>
+      <div>
+        <ul class="htmlcss">
+          <li class="item4">css</li>
+          <li class="item5"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item6">html</li>
+          <li class="item7"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item8">bootstrap</li>
+          <li class="item9"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item10">ruby</li>
+        </ul>
+        <a href="#" class="button">See Project</a>
+      </div>
+    </div>
+    <div class="grid-container">
+      <div class="item1"></div>
+      <div class="item2"><div class="bgr2"><img src="images/Ellipse 21@2x.png" alt="Ellipse"></div><h4>Multi-Post Stories</h4></div>
+      <div class="item3"> 
+      <p>
+        A daily selection of privately<br> personalized reads; no accounts<br>
+         or sign-ups required. has been the<br> industry's standard dummy text<br>
+          ever since the 1500s, when an<br> unknown printer took a standard<br> dummy text.
+      </p> 
+      </div>
+      <div>
+        <ul class="htmlcss">
+          <li class="item4">css</li>
+          <li class="item5"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item6">html</li>
+          <li class="item7"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item8">bootstrap</li>
+          <li class="item9"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item10">ruby</li>
+        </ul>
+        <a href="#" class="button">See Project</a>
+      </div>
+    </div>
+    <div class="grid-container">
+      <div class="item1"></div>
+      <div class="item2"><h4>Multi-Post Stories</h4><div class="bgr3"><img src="images/download__1_-removebg-preview.png" alt="Ellipse"></div></div>
+      <div class="item3">
+      <p>
+        A daily selection of privately<br> personalized reads; no accounts<br>
+         or sign-ups required. has been the<br> industry's standard dummy text<br>
+          ever since the 1500s, when an<br> unknown printer took a standard<br> dummy text.
+      </p>   
+      </div>
+      <div>
+        <ul class="htmlcss">
+          <li class="item4">css</li>
+          <li class="item5"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item6">html</li>
+          <li class="item7"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item8">bootstrap</li>
+          <li class="item9"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item10">ruby</li>
+        </ul>
+        <a href="#" class="button">See Project</a>
+      </div>
+    </div>
+  </div>
```

Comment:
- [ ] While you did a good effort wrapping the elements with `<section>` tag, as the previous code reviewer asked, please include all the projects cards into that `<section>` tag and align all the projects cards using grid :+1: